ocamldoc/Makefile

```diff
@@ -1,254 +0,0 @@
-#**************************************************************************
-#*                                                                        *
-#*                                 OCaml                                  *
-#*                                                                        *
-#*             Maxence Guesdon, projet Cristal, INRIA Rocquencourt        *
-#*                                                                        *
-#*   Copyright 2001 Institut National de Recherche en Informatique et     *
-#*     en Automatique.                                                    *
-#*                                                                        *
-#*   All rights reserved.  This file is distributed under the terms of    *
-#*   the GNU Lesser General Public License version 2.1, with the          *
-#*   special exception on linking described in the file LICENSE.          *
-#*                                                                        *
-#**************************************************************************
-
-ROOTDIR = ..
-
-include $(ROOTDIR)/Makefile.common
-include $(ROOTDIR)/Makefile.best_binaries
-
-OCAMLC = $(BEST_OCAMLC) $(STDLIBFLAGS)
-OCAMLOPT = $(BEST_OCAMLOPT) $(STDLIBFLAGS)
-
-# For installation
-##############
-
-programs := ocamldoc ocamldoc.opt
-
-OCAMLDOC_LIBCMA=odoc_info.cma
-OCAMLDOC_LIBCMI=odoc_info.cmi
-OCAMLDOC_LIBCMXA=odoc_info.cmxa
-OCAMLDOC_LIBA=odoc_info.$(A)
-
-OCAMLDOC_LIBMLIS=$(addsuffix .mli,\
-  odoc_dep odoc_dot odoc_extension odoc_html odoc_info odoc_latex \
-  odoc_latex_style odoc_man odoc_messages odoc_ocamlhtml odoc_parameter \
-  odoc_texi odoc_text_lexer odoc_to_text odoc_type odoc_value)
-OCAMLDOC_LIBCMIS=$(OCAMLDOC_LIBMLIS:.mli=.cmi)
-OCAMLDOC_LIBCMTS=$(OCAMLDOC_LIBMLIS:.mli=.cmt) $(OCAMLDOC_LIBMLIS:.mli=.cmti)
-
-ODOC_TEST=odoc_test.cmo
```

Comment:
This doesn't appear to have been merged? If I read correctly, the file was always built before?